frontend/configuration.properties.js

```diff
@@ -307,6 +307,14 @@
     // a tooltip containing the full name will be added to the component.
     maxHierarchicalSegmentLength: 0,
   },
+  
+  // These properties configure the tags/options shown for each of the list search results
+  SearchResultTags: {
+	  // Whether to show an option to add/view comments for a search result. False by default.
```

Comment:
Code style: Indentation is off
```suggestion
  // Whether to show an option to add/view comments for a search result. False by default.
```